src/http-spec.ts

```diff
@@ -7,8 +7,58 @@ import { IServer } from './servers';
  * HTTP Service
  */
 
+interface IOAuthFlow {
+  authorizationUrl: string;
+  tokenUrl: string;
+  refreshUrl?: string;
+  scopes: Dictionary<string>;
+}
+
+type SecurityDefinition =
+  | {
+      description?: string;
+    }
+  | {
+      type: 'apiKey';
+      name: string;
+      in: 'query' | 'header' | 'cookie';
+    }
+  | {
+      type: 'openIdConnect';
+      openIdConnectUrl: string;
+    }
+  | {
+      type: 'http';
+      scheme: string;
+      bearerFormat?: string;
+    }
+  | {
+      type: 'oauth2';
+      flows: {
+        implicit: IOAuthFlow;
+        password: IOAuthFlow;
+        clientCredentials: IOAuthFlow;
+        authorizationCode: IOAuthFlow;
+      };
+    };
+
 export interface IHttpService extends INode {
   name: string;
+  description?: string;
```

Comment:
Description is already on `INode`. Node `tags` prop on `INode` as well - we'll definitely want to populate that with the oas2/oas3 root tag info.

Reply:
Yeah I noticed the tags, didn't see the description. You can see how in crunch mode I am :trollface: